sycl/doc/GetStartedGuide.md

```diff
@@ -331,7 +331,7 @@ AOT compiler for each device type:
 #### CPU
 
 * CPU AOT compiler `opencl-aot` is enabled by default. For more, see
-[opencl-aot documentation](../../opencl-aot/README.md).
+[opencl-aot documentation](../../opencl/README.md).
```

Comment:
Mb it's better to move opencl-aot/ folder to opencl/opencl-aot/ ? The directory structure will be aligned with already existed internal opencl folder structure.

Reply:
Ok.